features/bicycle/ferry.feature

```diff
@@ -64,6 +64,6 @@ Feature: Bike - Handle ferry routes
             | abcd  |         | ferry | yes     | 1:00     |
 
         When I route I should get
-            | from | to | route     | time  |
-            | a    | d  | abcd,abcd | 3600s |
-            | d    | a  | abcd,abcd | 3600s |
+            | from | to | route     | time      |
+            | a    | d  | abcd,abcd | 3600s +-5 |
```

Comment:
Here is not an accuracy issue, but the expectation change: now turn durations are added to `time`. I think it is better to change the test to
```
     | x    | y  | xa,abcd,yd,yd | 3672s |
     | y    | x  | yd,abcd,xa,xa | 3669.8s |
```